frontend/TestRun/ResultsTab.svelte

```diff
@@ -337,9 +318,11 @@
                                 <tr>
                                     <td>{row}</td>
                                     {#each result.columns as col}
-                                        <td class="{ResultCellStatusStyleMap[result.table_data[row][col.name]?.status || 'NULL']}">
-                                            {formatValue(result.table_data[row][col.name]?.value, result.table_data[row][col.name]?.type || "NULL")}
-                                        </td>
+                                        {#key result.table_data[row][col.name]}
+                                            <td class="{ResultCellStatusStyleMap[result.table_data[row][col.name]?.status || 'NULL']}">
+                                                <Cell cell={result.table_data[row][col.name]} bind:selectedScreenshot/>
```

Comment:
This component seems to be missing from PR.

Reply:
true, updated.